chia/wallet/did_wallet/did_wallet.py

```diff
@@ -10,6 +10,7 @@
 
 from blspy import AugSchemeMPL, G1Element, G2Element
 
+import chia.wallet.singleton
```

Comment:
I'm not a huge fan of this, can we either throw the other singleton functions that we need into the import below or import this `as singleton`?

Reply:
Oh, sure.